### PR TITLE
Removed duplicacy and unnecesary logs for meilix build logs - (Solves- #440)

### LINKED ISSUE #440: Meilix-check script is called twice during the build
- Issue type: Bug report 

### Short description

Currently the output of the meilix build is real noisy,
One of the reasons is the execution of meilix-check script being operated twice
Meilix check operates once when all the bash scripts copied in chroot is executed,
`chmod +x ./*.sh
./*.sh
`
Next in the end of the script:
`chmod +x meilix_check.sh
./meilix_check.sh`

### PR DESCRIPTION
**Pull request by pranav1698**
### Short description
This removes duplicacy and unnecessary logs for the meilix travis builds, 
Meilix check is executes once and some debugs logs are now removed

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #440

### CODE REVIEW

File: scripts/chroot_reconst.sh [hunk 1]
```diff
@@ -94,16 +90,18 @@ dpkg -i systemlock_0.1-1_all.deb
 # /usr/share/applications/lxqt-about.desktop
 # or ~/.local/usr/share override (in skel!)
 # set the option NoDisplay=true
-cat /usr/share/applications/lxqt-about.desktop
```
Comment (pranav1698) [inline scripts/chroot_reconst.sh]: This was used for removing the about lxqt entry in the menu,
Can be removed

File: scripts/chroot_reconst.sh [hunk 2]
```diff
@@ -70,10 +70,6 @@ echo "deb https://download.sublimetext.com/ apt/stable/" | tee /etc/apt/sources.
 apt-get -qq update
 apt-get -qq -y install sublime-text
 
-# Installing related softwares
```
Comment (tabesin) [inline scripts/chroot_reconst.sh]: why are you removing this?

File: scripts/chroot_reconst.sh (hunk 1 above)
Comment (tabesin) [reply scripts/chroot_reconst.sh]: yes

File: scripts/chroot_reconst.sh [hunk 3]
```diff
@@ -119,10 +117,6 @@ rm -f plymouth-theme-meilix-text_1.0-2_all.deb
 rm -f meilix-default-theme_1.0-2_all.deb
 rm -f systemlock_0.1-1_all.deb
 
-# Meilix Check Skript
-chmod +x meilix_check.sh
-./meilix_check.sh
-
```
Comment (tabesin) [inline scripts/chroot_reconst.sh]: Comment this out if you want less verbosity. Don't delete.

Comment (xeon-zolt) [reply scripts/chroot_reconst.sh]: yea would be helpful in future for debugging if we required it by uncommenting it

Comment (pranav1698) [reply scripts/chroot_reconst.sh]: Ok, will do the same

File: scripts/chroot_reconst.sh (hunk 2 above)
Comment (pranav1698) [reply scripts/chroot_reconst.sh]: I have not removed it just executed in the bottom of the script.

File: scripts/chroot_reconst.sh (hunk 3 above)
Comment (pranav1698) [reply scripts/chroot_reconst.sh]: It would also be better if we comment out copying the meilix-check script into the chroot

**Comment by xeon-zolt** [comment]
I will wait for @tabesin  cause its his changes and he knows better than me

**Pull request merged**